modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -187,15 +218,46 @@ A few changes have been made to `Pill` to ensure proper accessibility and styles
 - `<Pill.IconButton/>` no longer has a default `aria-label="remove"`. You *must* provide an `aria-label` for `<Pill.IconButton/>` to ensure proper accessibility. Our examples have been updated to reflect this change.
 - `<Pill.Label/>` is a *required* element when using other sub-components like `<Pill.Icon/>` to ensure that the label truncates correctly.
```

Comment:
We're unfortunately inconsistent on "subcomponent" in Canvas Kit, but it is [a defined word](https://dictionary.cambridge.org/us/dictionary/english/subcomponent).

```suggestion
- `<Pill.Label/>` is a *required* element when using other subcomponents such as `<Pill.Icon/>` to ensure that the label truncates correctly.
```